app/components/login.js

```diff
@@ -0,0 +1,33 @@
+import Component from '@glimmer/component';
+import { tracked } from '@glimmer/tracking';
+import { AUTH } from '../constants/urls';
+import { service } from '@ember/service';
+
+export default class LoginComponent extends Component {
+  @service router;
+  @service fastboot;
+  @service login;
+  @tracked authURL = this.generateAuthURL();
```

Comment:
where is this getting used? and why we need tracking here?

Reply:
it is not being used, i used existing github auth code from header.js  and overlooked it...
auth links we'll be fetched from AUTH_URL below it.
i'll make sure to remove it.